testsuite/pytests/test_compartmental_model.py

```diff
@@ -372,33 +372,34 @@ def create_2tdend_4comp(dt=0.1):
     aa[8, 6] = -DP[7]['g_C'] / 2.
     aa[8, 8] = DP[7]['C_m'] / dt + DP[7]['g_L'] / 2. + DP[7]['g_C'] / 2.
 
-    bb[0] = SP['C_m'] / dt * SP['e_L'] + SP['g_L'] * SP['e_L'] / 2. - \
-        DP[0]['g_C'] * (SP['e_L'] - DP[0]['e_L']) / 2. - \
-        DP[4]['g_C'] * (SP['e_L'] - DP[4]['e_L']) / 2.
+    v_init = -70.
 
-    bb[1] = DP[0]['C_m'] / dt * DP[0]['e_L'] + DP[0]['g_L'] * DP[0]['e_L'] / 2. - \
-        DP[0]['g_C'] * (DP[0]['e_L'] - SP['e_L']) / 2. - \
-        DP[1]['g_C'] * (DP[0]['e_L'] - DP[1]['e_L']) / 2.
-    bb[2] = DP[1]['C_m'] / dt * DP[1]['e_L'] + DP[1]['g_L'] * DP[1]['e_L'] / 2. - \
-        DP[1]['g_C'] * (DP[1]['e_L'] - DP[0]['e_L']) / 2. - \
-        DP[2]['g_C'] * (DP[1]['e_L'] - DP[2]['e_L']) / 2. - \
-        DP[3]['g_C'] * (DP[1]['e_L'] - DP[3]['e_L']) / 2.
-    bb[3] = DP[2]['C_m'] / dt * DP[2]['e_L'] + DP[2]['g_L'] * DP[2]['e_L'] / 2. - \
-        DP[2]['g_C'] * (DP[2]['e_L'] - DP[1]['e_L']) / 2.
-    bb[4] = DP[3]['C_m'] / dt * DP[3]['e_L'] + DP[3]['g_L'] * DP[3]['e_L'] / 2. - \
-        DP[3]['g_C'] * (DP[3]['e_L'] - DP[1]['e_L']) / 2.
+    bb[0] = SP['C_m'] / dt * v_init - \
+        SP['g_L'] * (v_init / 2. - SP['e_L'])
+
+    bb[1] = DP[0]['C_m'] / dt * v_init - \
+        DP[0]['g_L'] * (v_init / 2. - DP[0]['e_L'])
+
+    bb[2] = DP[1]['C_m'] / dt * v_init - \
+        DP[1]['g_L'] * (v_init / 2. - DP[1]['e_L'])
 
-    bb[5] = DP[4]['C_m'] / dt * DP[4]['e_L'] + DP[4]['g_L'] * DP[4]['e_L'] / 2. - \
-        DP[4]['g_C'] * (DP[4]['e_L'] - SP['e_L']) / 2. - \
-        DP[5]['g_C'] * (DP[4]['e_L'] - DP[5]['e_L']) / 2.
-    bb[6] = DP[5]['C_m'] / dt * DP[5]['e_L'] + DP[5]['g_L'] * DP[5]['e_L'] / 2. - \
-        DP[5]['g_C'] * (DP[5]['e_L'] - DP[4]['e_L']) / 2. - \
-        DP[6]['g_C'] * (DP[5]['e_L'] - DP[6]['e_L']) / 2. - \
-        DP[7]['g_C'] * (DP[5]['e_L'] - DP[7]['e_L']) / 2.
-    bb[7] = DP[6]['C_m'] / dt * DP[6]['e_L'] + DP[6]['g_L'] * DP[6]['e_L'] / 2. - \
-        DP[6]['g_C'] * (DP[6]['e_L'] - DP[5]['e_L']) / 2.
-    bb[8] = DP[7]['C_m'] / dt * DP[7]['e_L'] + DP[7]['g_L'] * DP[7]['e_L'] / 2. - \
-        DP[7]['g_C'] * (DP[7]['e_L'] - DP[5]['e_L']) / 2.
+    bb[3] = DP[2]['C_m'] / dt * v_init - \
+        DP[2]['g_L'] * (v_init / 2. - DP[2]['e_L'])
+
+    bb[4] = DP[3]['C_m'] / dt * v_init - \
+        DP[3]['g_L'] * (v_init / 2. - DP[3]['e_L'])
+
+    bb[5] = DP[4]['C_m'] / dt * v_init - \
+        DP[4]['g_L'] * (v_init / 2. - DP[4]['e_L'])
+
+    bb[6] = DP[5]['C_m'] / dt * v_init - \
+        DP[5]['g_L'] * (v_init / 2. - DP[5]['e_L'])
+
+    bb[7] = DP[6]['C_m'] / dt * v_init - \
+        DP[6]['g_L'] * (v_init / 2. - DP[6]['e_L'])
+
+    bb[8] = DP[7]['C_m'] / dt * v_init - \
+        DP[7]['g_L'] * (v_init / 2. - DP[7]['e_L'])
```

Comment:
I don't see why these lines need breaking. They don't seem to be longer than 120 characters. The same applies above.